in_progress/mission-statement.md

```diff
@@ -0,0 +1,53 @@
+# Mission Statement
+
+## Goals
+
+We want to make single cell data analysis as accesible as possible. This means
+
+* Keeping data open, prioritizing interoperable data formats over serialized objects
+* Diversity, we believe in removing barriers to participation in research, including economic factors and institutional culture. Our tools will have the most impact if they are developed by and for a diverse community.
+* Transparency, we do our work as openly as possible and welcome our community to join in the decision making process
+* Simplicity – Invent as little as possible and work with standard pydata types. Our users already have to be experts in multiple-fields. Lets not raise the bar for usage higher.
+
+## Abstract
+
+Scanpy and accessory packages for single-cell data analysis in Python have seen a strong and steady increase in interest since its first publication.
+The ecosystem has outgrown a single package and modality, so we've formed a new organization, the **`scverse`**, to handle this.
+
+The `scverse` is a community-based ecosystem of tools with users and developers across many organizations.
+For the ecosystem to be useful and grow, it’s essential that the core analytic tools are well made and continue to be maintained. Frequently, tools are made by small groups within individual labs, and can quickly fall into disrepair once a grad student leaves. To some extent, this is unavoidable and is the nature of academic code. For commonly used, important tools this needs to be avoided.
+
+The scverse constitutes an environment of tools which is deliberately not unified under a single package, but instead a consortium of core analytic tools with shared maintanance resposibilities.
+Tools within this environment that are considered "core" for sharing data between tools or are required for fundamental analyses of single cell datatypes are placed under shared maintainance in the `scverse` github organization.
+
+Workflows in this environment are heterogeneous and this core can be supplemented with analysis specific accessory packages.
+This document lays out our vision and processes around the project's community in a transparent and accessible way.
+We are committed to keeping this community open to third-party contributions and usage and are committed to acknowledging contributions.
+This white paper may serve as an entry point for third parties into the structure of the scverse environment.
+
+## User engagement
+
+The ecosystem centred on `scverse` core packages consists of python packages which are documented on readthedocs and commonly accompanied by scientific publications.
```

Comment:
```suggestion
The ecosystem centered on `scverse` core packages consists of Python packages that are documented on readthedocs and commonly accompanied by scientific publications.
```

I don't understand what this is trying to say

Reply:
`that are documented on readthedocs` -> `that follow best software engineering practices`

Reply:
Can we even make such a statement about the "ecosystem"? We can guarantee best practice for "core" packages, but not necessarily the ecosystem?

Reply:
Good point. Agree with @grst